From: SeongJae Park <sj@kernel.org>
Date: Thu, 19 Dec 2024 09:59:55 -0800
Subject: [PATCH] mm/damon/sysfs-schemes: document sz_filter_passed on
 damos_sysfs_populate_region_dir() kernel-doc comment

Absence of the documentation causes the reported build warning.  Fix it
by documenting it.

Reported-by: kernel test robot <lkp@intel.com>
Closes: https://lore.kernel.org/oe-kbuild-all/202412191225.f6bEMRT2-lkp@intel.com/
Signed-off-by: SeongJae Park <sj@kernel.org>
---
 mm/damon/sysfs-schemes.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/mm/damon/sysfs-schemes.c b/mm/damon/sysfs-schemes.c
index 515d27b4e503..7462319dd30b 100644
--- a/mm/damon/sysfs-schemes.c
+++ b/mm/damon/sysfs-schemes.c
@@ -2124,6 +2124,7 @@ void damon_sysfs_schemes_update_stats(
  * @r:			DAMON region to populate the directory for.
  * @s:			Corresponding scheme.
  * @total_bytes_only:	Whether the request is for bytes update only.
+ * @sz_filter_passed:	Bytes of @r that passed filters of @s.
  *
  * Called from DAMOS walk callback while holding damon_sysfs_lock.
  */
-- 
2.39.5

